pages/[lng]/register/index.tsx

```diff
@@ -1,5 +1,5 @@
 /* library package */
-import { FC, useState } from 'react'
+import { FC, useRef, useState } from 'react'
```

Comment:
```suggestion
import {
  FC,
  useRef,
  useState
} from 'react'
```

legenda mengatakan jikalau improt-nya ada lebih dari 2 (dua), maka dikasih enter seperti contoh di atas.